src/native/corehost/apphost/static/singlefilehost_OSXexports.src

```diff
@@ -1,11 +0,0 @@
-; Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Woops. This file should not be deleted. Rather renamed to src/native/corehost/apphost/static/singlefilehost_freebsdexports.src and you'd need to add the other two exports that got deleted in the next file.

Reply:
Woops indeed! Thanks for catching that, fixed :)